src/context/online.rs

```diff
@@ -220,4 +220,12 @@ impl AudioContext {
         let opts = node::MediaElementAudioSourceOptions { media_element };
         node::MediaElementAudioSourceNode::new(self, opts)
     }
+
+    /// Returns an [`AudioRenderCapacity`] instance associated with an AudioContext.
+    #[must_use]
+    pub fn render_capacity(&self) -> AudioRenderCapacity {
+        let (sender, receiver) = crossbeam_channel::bounded(1);
```

Comment:
Could we somehow reuse the sender and receiver (just clone) so that we have only one sender in audio thread? This would allow to:
- send only 1 message between threads, even if `context.render_capacity()` has been called several time
- avoid the `Vec` and the loop in render thread